app/static/lib/conp-react/src/DatasetElement/DownloadModalWindowElement/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import * as ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+
+const DownloadModalWindowElement = (props) => {
+  const { authorized, imagePath, ...element } = props;
```

Comment:
I think only `element.size` and `element.zipLocation` are used, so we could just pass those two directly and use a lot fewer props for this component.